docs/content/java-notes.md

```diff
@@ -15,11 +15,54 @@ application, `lcm.jar` must be in your Java classpath.
 On Linux, OS/X, and other UNIX-like systems, `lcm.jar` is typically built
 automatically and installed along with the rest of LCM.  The exact location
 depends on the operating system and any configuration parameters, but it can
-often be found installed in <tt>/usr/local/share/java/</tt>. 
+often be found installed in `/usr/local/share/java/`. 
 
 Separately, `lcm.jar` can also be found in the `lcm-java` subdirectory of the
 source distribution after compiling LCM from source.
 
+A `lcm-sources.jar` is also built and installed. This can provide IDE Javadoc integration.
+
+### Gradle project
+
+An example of using LCM in a Gradle project. Assuming you have placed `my-lcm-types.jar` in `./lib`:
+
+```gradle
+// build.gradle
+String osName = System.getProperty("os.name").toLowerCase();
+project.logger.lifecycle(osName)
+repositories { 
+    mavenCentral()
+
+    flatDir {
+        if (osName.contains("linux")) {        
+            dirs '/usr/local/share/java'
+        } // else TODO
+        dirs 'libs'
+    }
+}
+
+dependencies {
+    implementation ':lcm'
+    implementation ':my-lcm-types'
+}    
+```
+
+> Note: `flatDir` is preferred over `implementation files('/usr/local/share/java/lcm.jar')`. Both work, but IDEs may not be able to find `lcm-sources.jar` with the `files` approach.
+
+### Ant project
+TODO. Know what to do? Please contribute.
+
+### Maven project
+TODO. Know what to do? Please contribute.
```

Comment:
I'd rather not leave TODOs rendering in the official documentation, but I like the request for contributions. What do you think about turning this into a misc. section at the end that asks for contributions for any missing IDEs or systems?